Add support for dflydev/fig-cookies ^3.0

MichaelGooden opened the pull request:
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

`fig-cookies` bumped their major version, although no BC breaks were documented or visible in the upstream project changesets.

See also: https://github.com/mezzio/mezzio-swoole/issues/54#issuecomment-765381312

MichaelGooden committed:
Add support for dflydev/fig-cookies ^3.0

Signed-off-by: Michael Gooden <me@michaelgooden.net>

MichaelGooden commented:
Failing test seems unrelated to changes:
```
FILE: test/SessionMiddlewareTest.php

----------------------------------------------------------------------

FOUND 4 ERRORS AFFECTING 4 LINES

----------------------------------------------------------------------

 53 | ERROR | [x] Class \PHPUnit\Framework\MockObject\MockObject

    |       |     should not be referenced via a fully qualified

    |       |     name, but via a use statement.

 64 | ERROR | [x] Class \PHPUnit\Framework\MockObject\MockObject

    |       |     should not be referenced via a fully qualified

    |       |     name, but via a use statement.

 67 | ERROR | [x] Class \PHPUnit\Framework\MockObject\MockObject

    |       |     should not be referenced via a fully qualified

    |       |     name, but via a use statement.

 75 | ERROR | [x] Class \PHPUnit\Framework\MockObject\MockObject

    |       |     should not be referenced via a fully qualified

    |       |     name, but via a use statement.

----------------------------------------------------------------------

PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY

----------------------------------------------------------------------

FILE: test/SessionTest.php

----------------------------------------------------------------------

FOUND 1 ERROR AFFECTING 1 LINE

----------------------------------------------------------------------

 118 | ERROR | [x] Class \stdClass should not be referenced via a

     |       |     fully qualified name, but via a use statement.

----------------------------------------------------------------------

PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY

----------------------------------------------------------------------
```

froschdesign commented:
@MichaelGooden 
> Failing test seems unrelated to changes:

If you can create an issue report or pull request for these problems that would be fantastic. Thanks in advance!

MichaelGooden commented:
> @MichaelGooden
> 
> > Failing test seems unrelated to changes:
> 
> If you can create an issue report or pull request for these problems that would be fantastic. Thanks in advance!

Please see #19 (those tests failing for a _different_ reason :D)

weierophinney committed:
docs: remove CHANGELOG.md

Changelogs will be kept in the milestone description and propagated to the tag and release notes.

Signed-off-by: Matthew Weier O'Phinney <matthew@weierophinney.net>
